pkgs/applications/editors/emacs/make-emacs.nix

```diff
@@ -13,7 +13,7 @@
   Xaw3d,
   acl,
   alsa-lib,
-  apple-sdk,
+  apple-sdk_14,
```

Comment:
@juuyokka is `apple-sdk_14` the minimum sdk version that is needed for this?

Reply:
It has additional functionality on newer SDKs. There is no harm in using a newer SDK for a correctly-written application that does runtime  availability checks before using newer APIs, and this appears to do so. However, macOS 14 will be the baseline for 25.11 anyway, so it may not be worth explicitly specifying until they implement optional functionality for newer versions.